src/misc/nyaize.ts

```diff
@@ -3,7 +3,9 @@ export function nyaize(text: string): string {
 		// ja-JP
 		.replace(/な/g, 'にゃ').replace(/ナ/g, 'ニャ').replace(/ﾅ/g, 'ﾆｬ')
 		// en-US
-		.replace(/morning/gi, 'mornyan').replace(/everyone/gi, 'everynyan')
+		.replace(/Na/g, 'Nya').replace(/na/g, 'nya')
+		.replace(/Morning/g, 'Mornyan').replace(/morning/g, 'mornyan')
+		.replace(/Everyone/g, 'Everynyan').replace(/everyone/g, 'everynyan')
```

Comment:
How about look like this?

```suggestion
		.replace(/(?<=n)a/gi, x => x === 'A' ? 'YA' : 'ya')
		.replace(/(?<=morn)ing/gi, x => x === 'ING' ? 'YAN' : 'yan')
		.replace(/(?<=every)one/gi, x => x === 'ONE' ? 'NYAN' : 'nyan')
```

It can replace all of an uppercase word, a titlecase word, and a lowercase word per one call.

Reply:
That's great, though if you have "nAme" (or similar cases with the other replacements) you will get a result of "nYAme". If that's okay then I'd commit your changes.

Reply:
I'm sorry for my slow response. I would rather want to get a result of "nYAme" when we have "nAme" or etc. (@syuilo said "The Cat mode is a joke feature that allows you to enjoy the inexcusability of be turned "な (na)" into "にゃ (nya)" forcely.)

Reply:
No problem. Alright then, i'm going to commit your suggestion and then this should be ready to merge. 👍